docs/src/rules/lines-around-comment.md

```diff
@@ -717,6 +718,35 @@ foo();
 
 :::
 
+### afterHashbangComment
+
+Examples of **incorrect** code for this rule with the `{ "afterHashbangComment": true }` option:
+
+::: incorrect
+
+```js
+#!foo
+var day = "great"
+
+/*eslint lines-around-comment: ["error", { "afterHashbangComment": true }] */
```

Comment:
All of our examples have the `/*eslint rule: []*/` comment at the top, will this still work if we move it to the top? I believe the Hashbang comment should be present at the top of the file that's why you have moved it to bottom.

Reply:
Hashbang comments in second line will result in a syntax error, so `/*eslint rule: []*/` cannot be put at top.